avocado_vt/test.py

```diff
@@ -203,7 +203,7 @@ def verify_background_errors(self):
         except Queue.Empty:
             pass
         else:
-            raise exc[1], None, exc[2]
+            raise exc[1].with_traceback(exc[2])
```

Comment:
need introduce six module to keep compatable:
-raise E, V, T
+mport six
+six.reraise(E, V, T)